python/tvm/relay/op/contrib/cmsisnn.py

```diff
@@ -229,17 +229,18 @@ def qnn_max_pool2d_pattern():
     def check_qnn_max_pool2d(pattern):
         """Check if max pool2d is supported by CMSIS-NN."""
         output = pattern
-        input_var = _find_last(pattern)
+        input_op = None
```

Comment:
We can remove this as it's always set later.

Reply:
ACK